docs/platform/concepts/permissions.md

```diff
@@ -8,58 +8,69 @@ To give users access to projects and services in your organizations, you grant t
   group of resources.
 * **Roles**: Sets of permissions that you can assign to a principal.
 
-Principals can be:
-* [Organization users](/docs/platform/howto/manage-org-users)
-* [Application users](/docs/platform/concepts/application-users)
-* [Groups](/docs/platform/howto/manage-groups)
+Principals are
+[organization users](/docs/platform/howto/manage-org-users),
+[application users](/docs/platform/concepts/application-users),
+and [groups](/docs/platform/howto/manage-groups).
 
+You can grant access to principals at the organization and project level.
 You can
-[grant access to principals at the project level](/docs/platform/howto/manage-permissions).
-You can also [add users to services](/docs/platform/howto/create_new_service_user).
+[add users to services](/docs/platform/howto/create_new_service_user).
```

Comment:
Could you clarify the purpose of including 'add users to services' here? It feels slightly disconnected from the previous sentence. Is it meant to complement granting access or highlight a separate action?